### ISSUE TITLE
fix CRAN check issues

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/SpaDES.core#1]
See https://cran.r-project.org/web/checks/check_results_SpaDES.core.html

```r
* checking dependencies in R code ... NOTE
Namespaces in Imports field not imported from:
  ‘CircStats’ ‘RandomFields’ ‘grDevices’ ‘sp’
  All declared Imports should be used.
```

**Comment by achubaty** [comment]
- `CircStats` is used for documentation (via `@inheritParams`)
- `RandomFields` and `sp` are used by the sample modules
- `grDevices` isn't being used anywhere and can likely be removed

**Comment by achubaty** [comment]
merged into master branch with d3962f852a0fc12a7b20ced69a42a055f47600e2
